app/assets/javascripts/initializers/initializeTimeFixer.js

```diff
@@ -38,13 +42,13 @@ function initializeTimeFixer() {
 
   function updateLocalDate(dates) {
     var localDate;
-    for (var i = 0; i < dates.length; i++) {
+    for (var i = 0; i < dates.length; i += 1) {
       localDate = convertUtcDate(dates[i].dataset.datetime);
       dates[i].innerHTML = localDate;
     }
   }
 
-  function convertCalEvent(utc) {
+  function convertCalEvent(UTC) {
```

Comment:
I think a better refactoring would be to move the function, using "casing" not to trigger eslint rules isn't great...